.vscode/extensions.json

```diff
@@ -0,0 +1,15 @@
+{
+  // List of extensions which should be recommended for users of this workspace.
+  "recommendations": [
+    "golang.go",
+    "editorconfig.editorconfig",
+    "dbaeumer.vscode-eslint",
+    "esbenp.prettier-vscode",
+    "voorjaar.windicss-intellisense",
+    "johnsoncodehk.volar",
+    "johnsoncodehk.vscode-typescript-vue-plugin",
```

Comment:
From the [Readme](https://github.com/johnsoncodehk/volar/tree/master/extensions/vscode-typescript-vue-plugin#readme):

> Recommended [use take over mode instead of this plugin](https://github.com/johnsoncodehk/volar/discussions/471).